stats.json

```diff
@@ -2,10 +2,10 @@
   {
     "name": "Arabic",
     "code": "ar",
-    "phrases": "8282",
+    "phrases": "8287",
```

Comment:
Can you omit the changes to `stats.json` from this PR? They will happen automatically on build.